l10n_ro_payment_to_statement/models/sequence_mixin.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2015-2023 Deltatech
+# Copyright (C) 2022 NextERP Romania
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+from odoo.tools.safe_eval import safe_eval
+
+
+class SequenceMixin(models.AbstractModel):
+    _inherit = "sequence.mixin"
+
+    def _constrains_date_sequence(self):
+        get_param = self.env["ir.config_parameter"].sudo().get_param
+        override = safe_eval(
```

Comment:
Care e use case-ul pentru asta, adica odata activat nu mai respecta numerotarea, ceea ce nu e ok, cel putin pentru facturi.

Reply:
Este pentru cazul in care ai ceva ce seamana cu un an in seria facturilor (ex. XXX22/)